src/components/communitySuccess.tsx

```diff
@@ -9,7 +9,7 @@ export const CommunitySuccessPage = () => {
   const discordClientId = process.env.NEXT_PUBLIC_DISCORD_CLIENT_ID
 
   if (!discordClientId) {
-    toast.error(`Missing environment variable DISCORD_CLIENT_ID`)
+    toast.error(`Missing environment variable NEXT_PUBLIC_DISCORD_CLIENT_ID`)
```

Comment:
<img width="430" alt="Screenshot 2023-05-02 at 18 56 40" src="https://user-images.githubusercontent.com/9698363/235734883-25ae395b-6663-4267-a97e-7a61d8983d05.png">

Reply:
@nebs-dev please wait with requesting review until the code is working.